src/tests/conformance/post_handshake/validators.rs

```diff
@@ -55,3 +71,255 @@ async fn c015_TM_VALIDATOR_LIST_COLLECTION_node_should_send_validator_list() {
+    manifest[2] = ((sequence >> 16) & 0xff) as u8;
+    manifest[3] = ((sequence >> 8) & 0xff) as u8;
+    manifest[4] = (sequence & 0xff) as u8;
+    let mut i = 5;
+
+    // serialize public key
+    manifest[i] = 0x71; // field code 1 for "PublicKey"
+    manifest[i + 1] = PUBLIC_KEY_SIZE as u8;
+    i += 2;
+    manifest[i..i + PUBLIC_KEY_SIZE].clone_from_slice(public_key.as_slice());
+    i += PUBLIC_KEY_SIZE;
+
+    // serialize signing public key
+    manifest[i] = 0x73; // field code 3 for "SigningPubKey"
+    manifest[i + 1] = PUBLIC_KEY_SIZE as u8;
+    i += 2;
+    manifest[i..i + PUBLIC_KEY_SIZE].clone_from_slice(signing_pub_key.as_slice());
+    manifest
+}
+
+fn create_final_manifest(
+    sequence: u32,
+    public_key: &Vec<u8>,
+    signing_pub_key: &Vec<u8>,
+    master_signature: &Vec<u8>,
+    signature: &Vec<u8>,
+) -> Vec<u8> {
+    let size = 5
+        + 2
+        + public_key.len()
+        + 2
+        + signing_pub_key.len()
+        + 3
+        + master_signature.len()
+        + 2
+        + signature.len();
+    let mut manifest: Vec<u8> = vec![0; size];
+    manifest[0] = 0x24;
+    manifest[1] = ((sequence >> 24) & 0xff) as u8;
+    manifest[2] = ((sequence >> 16) & 0xff) as u8;
+    manifest[3] = ((sequence >> 8) & 0xff) as u8;
+    manifest[4] = (sequence & 0xff) as u8;
+    let mut i = 5;
+
+    // serialize public key
+    manifest[i] = 0x71; // field code 1 for "PublicKey"
+    manifest[i + 1] = PUBLIC_KEY_SIZE as u8;
+    i += 2;
+    manifest[i..i + PUBLIC_KEY_SIZE].clone_from_slice(public_key.as_slice());
+    i += PUBLIC_KEY_SIZE;
+
+    // serialize signing public key
+    manifest[i] = 0x73; // field code 3 for "SigningPubKey"
+    manifest[i + 1] = PUBLIC_KEY_SIZE as u8;
+    i += 2;
+    manifest[i..i + PUBLIC_KEY_SIZE].clone_from_slice(signing_pub_key.as_slice());
+    i += PUBLIC_KEY_SIZE;
+
+    // serialize signature
+    manifest[i] = 0x76; // field code 6 for "Signature"
+    manifest[i + 1] = signature.len() as u8;
+    i += 2;
+    manifest[i..i + signature.len()].clone_from_slice(signature.as_slice());
+    i += signature.len();
+
+    // serialize master signature
+    manifest[i] = 0x70; // field code 18 for "MasterSignature"
+    manifest[i + 1] = 0x12;
+    manifest[i + 2] = master_signature.len() as u8;
+    i += 3;
+    manifest[i..i + master_signature.len()].clone_from_slice(master_signature.as_slice());
+    manifest
+}
+
+fn sign_buffer(secret_key: &SecretKey, buffer: &Vec<u8>) -> Vec<u8> {
+    let engine = Secp256k1::new();
+    let digest = create_sha512_half_digest(buffer);
+    let message = Message::from_slice(&digest).unwrap();
+    let sig = engine.sign_ecdsa(&message, secret_key);
+    let sigser = sig.serialize_der();
+    let sigb64 = base64::encode(sigser);
+    base64::decode(sigb64).expect("unable to decode a blob")
+}
+
+#[tokio::test]
+#[allow(non_snake_case)]
+async fn c026_TM_VALIDATOR_LIST_send_validator_list() {
```

Comment:
I think you may have missed updating the SPEC.md document.

Usually every test we have in `.rs` files has a description in the [SPEC.md](https://github.com/runziggurat/xrpl/blob/main/SPEC.md) document

Reply:
As much as I agree with keeping SPEC.md up to date, I think we can leave it until the test is finished. (Or maybe just put some 'placeholder' kind of comment there for now).